### PR TITLE
Update prometheus ingress

### PR DESCRIPTION
**Pull request by korjek**
## What does this PR change?
Fix the issue with Prometheus Ingress spec.

## Does this PR rely on any other PRs?
No
- 
- 

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
No impact

## Links to Issues or ZD tickets this PR addresses or fixes

- 
- 

## How was this PR tested?
With `helm update` for running cost-analyzer helm release.

## Have you made an update to documentation?
Not needed

### CODE REVIEW

**Comment by korjek** [comment]
guys, could we add this to 1.88.0 release as well?
This blocks us from upgrading.

**Comment by AjayTripathy** [comment]
Hi @korjek ... thank you so much for the PR. I'll get it reviewed and see if we can do a 1.88.1 patch release or just re-release 1.88.0

File: cost-analyzer/charts/prometheus/templates/server-ingress.yaml [hunk 1]
```diff
@@ -34,10 +36,19 @@ spec:
 {{ if $extraPaths }}
 {{ toYaml $extraPaths | indent 10 }}
 {{- end }}
+          {{- if $apiV1 }}
+          - path: /{{ rest $url | join "/" }}
+            backend:
+              service:
+                name: {{ $serviceName }}
+                port:
```
Comment (AjayTripathy) [inline cost-analyzer/charts/prometheus/templates/server-ingress.yaml:44]: Are we sure about this indent? Isn't port supposed to be outdented here?

**Review by AjayTripathy — CHANGES_REQUESTED** [review]

**Comment by AjayTripathy** [comment]
@korjek can you share a subset of the generated YAML file from `helm template` or the prometheus ingress yaml manifest from your testing  ? I'm not sure how this PR would work with the "port" block indented like that.

**Review by AjayTripathy — APPROVED** [review]

**Comment by AjayTripathy** [comment]
Hi @korjek just letting you know this should be in our staging build but we can't add it retroactively to 1.88.0 

https://github.com/kubecost/docs/blob/main/staging.md for instructions-- otherwise it will be in our next major release. Thanks.

**Pull request merged**